### PR TITLE
Fixes #1201 & #1199: Options - Tablet (Portrait) (Landscape) (Highfi) 

### LINKED ISSUE #1199: Options - Tablet (Landscape) (Highfi)
Mocks: (not ready yet)  

Implement high-fi UI for **Options** tablet landscape mode

**Target PR date**: 8 August 2020
**Target completion date**: 12 August 2020

### LINKED ISSUE #1201: Options - Tablet (Portrait) (Highfi)
Mocks: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/20631bfd-3756-47a1-9353-ac0a8e9615cd/  

Implement high-fi UI for **Options** tablet portrait mode

**Target PR date**: 8 August 2020
**Target completion date**: 12 August 2020

### PR DESCRIPTION
**Pull request by MohamedMedhat1998**
## Explanation

- Fixes #1199 
- Fixes #1201 
- Implemented high-fi UI for **OptionsActivity** for tablet
- Mocks: [land](https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/421aa1ca-594a-452e-8367-c990916a5fe6/) - [port](https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/20631bfd-3756-47a1-9353-ac0a8e9615cd/)
## Screenshots
screenshot-٢٠٢٠-٠٨-٣٠_٢١ ٣٩ ٥٠ ٠٠٧
screenshot-٢٠٢٠-٠٨-٣٠_٢١ ٣٩ ٤٨ ٢٣٢
screenshot-٢٠٢٠-٠٨-٣٠_٢١ ٣٩ ٤٦ ٢٦٢
screenshot-٢٠٢٠-٠٨-٣٠_٢١ ٣٩ ٣٩ ٥٩٦
screenshot-٢٠٢٠-٠٨-٣٠_٢١ ٣٩ ٣٧ ٠٣٦
screenshot-٢٠٢٠-٠٨-٣٠_٢١ ٣٩ ٣٢ ٨٠٢

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/java/org/oppia/app/options/OptionsFragmentPresenter.kt [hunk 1]
```diff
@@ -122,23 +134,43 @@ class OptionsFragmentPresenter @Inject constructor(
...
+      )
+    )
+  }
+
+  private fun getSelectedFragmentIndex(selectedFragment: String): Int {
+    return when (selectedFragment) {
+      READING_TEXT_SIZE_FRAGMENT -> 0
+      APP_LANGUAGE_FRAGMENT -> 1
+      DEFAULT_AUDIO_FRAGMENT -> 2
+      else -> throw InvalidParameterException()
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/options/OptionsFragmentPresenter.kt]: Please write some error message too inside this exception like:
`InvalidParameterException ("Not a valid fragment in getSelectedFragmentIndex.")`

**Review by rt4914 — CHANGES_REQUESTED** [review]
Suggested one nit change. Else looks good to me. Thanks.

File: app/src/main/java/org/oppia/app/options/OptionsFragmentPresenter.kt (hunk 1 above)
Comment (MohamedMedhat1998) [reply app/src/main/java/org/oppia/app/options/OptionsFragmentPresenter.kt]: Done

**Review by rt4914 — APPROVED** [review]
LGTM

**Pull request merged**